lib/tsserver.js

```diff
@@ -92012,7 +92012,7 @@ var ts;
             };
             Logger.prototype.close = function () {
                 if (this.fd >= 0) {
-                    fs.close(this.fd);
```

Comment:
why not just add an empty handler instead?

Reply:
Because errors will be silently swallowed that way. I can of course do that but I do not feel like that is a good idea.

Reply:
> Because errors will be silently swallowed that way. 

that was the intention anyways.. we are killing the server, we want to try to write to the log if we can, but if we fail, not sure we have much recourse here, the process is dying anyways.